rust/package.json

```diff
@@ -17,8 +17,11 @@
     "typescript": "^4.3.5"
   },
   "scripts": {
-    "idl": "node test/idlToTs",
-    "test": "env MY_WALLET=$HOME/.config/solana/id.json ts-mocha -p ./tsconfig.json -t 1000000 test/*.ts"
+    "build": "anchor build --skip-lint",
```

Comment:
Let's not do `--skip-lint`. We need to actually fix those at some point, and this is going to hide the pain.